package.json

```diff
@@ -100,6 +100,7 @@
     "ts-jest": "^25.2.0",
     "ts-loader": "^6.2.1",
     "webpack": "^4.41.5",
+    "webpack-assets-manifest": "^3.1.1",
```

Comment:
Is there an advantage to using this over the plugin [mentioned in the docs](https://webpack.js.org/guides/output-management/#the-manifest)?

Reply:
We don't have any html in the project so I was a bit reluctant to revert back to having a wrapper document.

Reply:
Hmm, I'm not sure that hash url is that reliable, I was talking about [this plugin](https://github.com/danethurber/webpack-manifest-plugin), which is suggested at the bottom of that docs page.

Reply:
Oh I see, that's used by 1.1 million compared to 58K... I'll swap them over